galore/cli/galore_get_cs.py

```diff
@@ -54,28 +54,33 @@ def get_parser():
         Space-separated symbols for elements in material.""")
 
     parser.add_argument('--dataset', type=str,
-                        help='You can enter "Scofield" or "Yeh"')
+                        help=
+        """You can enter 'Scofield' or 'Yeh' """)
 
     return parser
 
 
 def run(energy, elements, dataset=None):
     cross_sections = galore.get_cross_sections(energy, elements, dataset)
-    logging = galore.cross_sections.cross_sections_info(cross_sections)
-    logging.info("Photoionisation cross sections per electron:")
+    if cross_sections is None:
+        pass
```

Comment:
Which scenario is this catching? Could there be a more helpful logging message in that case? Users don't like it when programs do nothing.